rlp/benches/rlp.rs

```diff
@@ -106,13 +80,13 @@ fn bench_stream_1000_empty_lists(b: &mut Bencher) {
 fn bench_decode_1000_values(b: &mut Bencher) {
 	let mut stream = RlpStream::new_list(1000);
 	for _ in 0..1000 {
-		stream.append(&U256::from(1));
+		stream.append(&1u64);
```

Comment:
This changes the amount of data tested, making it impossible to compare tests before/after this change.

In general I'm not sure why the dev dependencies can't depend on `primitive-types` (or any other code, be it `no_std` or not). I think the benchmark use the big uint types because it is such a common use case for the crate. What's the point of having the benchmarks be `no_std`?